controllers/blockdevice/blockdevice_controller.go

```diff
@@ -115,19 +89,19 @@ func (r *ReconcileBlockDevice) Reconcile(request reconcile.Request) (reconcile.R
 			// remove the finalizer string from BlockDevice resource
 			instance.Finalizers = util.RemoveString(instance.Finalizers, controllerutil.BlockDeviceFinalizer)
 			klog.Infof("Cleanup completed for %s", instance.Name)
-			err := r.updateBDStatus(openebsv1alpha1.BlockDeviceUnclaimed, instance)
+			err := r.updateBDStatus(apis.BlockDeviceUnclaimed, instance)
 			if err != nil {
 				klog.Errorf("Failed to mark %s as Unclaimed: %v", instance.Name, err)
 			}
 			r.recorder.Eventf(instance, corev1.EventTypeNormal, "BlockDeviceUnclaimed", "BD now marked as Unclaimed")
 		} else {
 			r.recorder.Eventf(instance, corev1.EventTypeNormal, "BlockDeviceCleanUpInProgress", "CleanUp is in progress")
 		}
-	case openebsv1alpha1.BlockDeviceClaimed:
+	case apis.BlockDeviceClaimed:
 		if !util.Contains(instance.GetFinalizers(), controllerutil.BlockDeviceFinalizer) {
 			// finalizer is not present, may be a BlockDevice claimed from previous release
 			instance.Finalizers = append(instance.Finalizers, controllerutil.BlockDeviceFinalizer)
-			err := r.client.Update(context.TODO(), instance)
+			err := r.Client.Update(context.TODO(), instance)
```

Comment:
Arent these update calls causing an issue? like conflicts when updating in the NDM daemon.